src/views/Homepage.elm

```diff
@@ -0,0 +1,20 @@
+module Homepage exposing (..)
+
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import HomepageHelper exposing (generateArrivalTimesTable)
+import ArrivalTypes exposing (Msg, StationArrivalTimes)
+
+
+index : StationArrivalTimes -> Html Msg
```

Comment:
What do you think about this name? I would opt for something a bit more descriptive. 

You could leverage the type signature in the name too, so something like this maybe?

```elm
show : StationArrivalTimes -> Html Msg
show stationArrivalTimes =
  // ..
```